PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/AzurePasswordFieldInput.java

```diff
@@ -24,23 +24,24 @@
 
 import com.microsoft.azure.toolkit.lib.common.form.AzureValidationInfo;
 import lombok.Getter;
-import lombok.Setter;
 import org.apache.commons.lang3.StringUtils;
 
 import javax.swing.*;
+import javax.swing.event.DocumentListener;
 
 public class AzurePasswordFieldInput implements AzureFormInputComponent<String> {
 
     private JPasswordField delegate;
 
     @Getter
-    @Setter
     private boolean passwordInitialized;
```

Comment:
what is used for? it seems should be a internal state(ie. it should not be public accessible)